<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupTab</class>
 <widget class="QWidget" name="SetupTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>455</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>612</width>
    <height>455</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="gbTCPIP">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>TCP/IP</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="loIP">
        <item>
         <widget class="QLabel" name="labelIP">
          <property name="text">
           <string>&amp;IP Address : </string>
          </property>
          <property name="buddy">
           <cstring>EditIP</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="EditIP"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="loPort">
        <item>
         <widget class="QLabel" name="labelPort">
          <property name="text">
           <string>&amp;Port : </string>
          </property>
          <property name="buddy">
           <cstring>EditPort</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="EditPort"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btnConnect">
          <property name="text">
           <string>Connect</string>
          </property>
          <property name="icon">
           <iconset resource="VfpResources.qrc">
            <normaloff>:/root/images/connect.png</normaloff>:/root/images/connect.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDisconnect">
          <property name="text">
           <string>Disconnect</string>
          </property>
          <property name="icon">
           <iconset resource="VfpResources.qrc">
            <normaloff>:/root/images/disconnect.png</normaloff>:/root/images/disconnect.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="gbDirectCommands">
        <property name="title">
         <string>Direct Commands</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="loDirectCommands">
           <item>
            <widget class="QComboBox" name="cbCommands">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnSend">
             <property name="text">
              <string>Send</string>
             </property>
             <property name="icon">
              <iconset resource="VfpResources.qrc">
               <normaloff>:/root/images/send.png</normaloff>:/root/images/send.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnReceive">
             <property name="text">
              <string>Receive</string>
             </property>
             <property name="icon">
              <iconset resource="VfpResources.qrc">
               <normaloff>:/root/images/receive.png</normaloff>:/root/images/receive.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gbInstrumentMessages">
        <property name="title">
         <string>Instrument Messages</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="loIntrumentMessages">
           <item>
            <widget class="QTextEdit" name="textInstrumentMessages"/>
           </item>
           <item alignment="Qt::AlignTop">
            <widget class="QPushButton" name="btnClear">
             <property name="text">
              <string>Clear</string>
             </property>
             <property name="icon">
              <iconset resource="VfpResources.qrc">
               <normaloff>:/root/images/clear.jfif</normaloff>:/root/images/clear.jfif</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>EditIP</tabstop>
  <tabstop>EditPort</tabstop>
  <tabstop>btnConnect</tabstop>
  <tabstop>btnDisconnect</tabstop>
  <tabstop>cbCommands</tabstop>
  <tabstop>btnSend</tabstop>
  <tabstop>btnReceive</tabstop>
  <tabstop>textInstrumentMessages</tabstop>
  <tabstop>btnClear</tabstop>
  <tabstop>btnConnect_2</tabstop>
  <tabstop>btnDisconnect_2</tabstop>
  <tabstop>EditIP_2</tabstop>
  <tabstop>EditPort_2</tabstop>
  <tabstop>cbCommands_2</tabstop>
  <tabstop>btnSend_2</tabstop>
  <tabstop>btnReceive_2</tabstop>
  <tabstop>textInstrumentMessages_2</tabstop>
  <tabstop>btnClear_2</tabstop>
 </tabstops>
 <resources>
  <include location="VfpResources.qrc"/>
 </resources>
 <connections/>
</ui>
